prime-router/src/main/kotlin/TransportType.kt

```diff
@@ -75,7 +75,7 @@ data class GAENTransportType
      * The UUID enables the GAEN system deduplicate notifications.
      * [uuidIV] is the HMAC initialization vector (aka key) in hex
      */
-    val apiUrl: String,
+    val apiUrl: String? = null,
```

Comment:
I'm not sure this is a safe change?  It get's used [here](https://github.com/CDCgov/prime-reportstream/blob/f714df13f63b18188f36507470765cb145a86228/prime-router/src/main/kotlin/transport/GAENTransport.kt#L230) and it seems like `post` expects it not to be null

Reply:
the urls in the settings are either "", or null if not set, when a sender/receiver is ready to join the pipeline, "customerStatus" will be set to "active", it's the onboarding's job to put a good url there before enabled the sender/receiver, this change won't impact real usage...

Reply:
I'm not necessarily following, wouldn't it make more sense to default it to `""`. A null value seems unsafe to me since the `POST` wants a string